sycl/include/sycl/handler.hpp

```diff
@@ -376,6 +376,14 @@ class __SYCL_EXPORT handler {
           std::shared_ptr<detail::queue_impl> PrimaryQueue,
           std::shared_ptr<detail::queue_impl> SecondaryQueue, bool IsHost);
 
+  /// Constructs SYCL handler from Graph.
+  ///
+  /// The hander will add the command-group as a node to the graph rather than
+  /// processing it straight away.
```

Comment:
```suggestion
  /// enqueueing it straight away.
```

I think this is a bit clearer.